src/cmps/tags/Tags.css

```diff
@@ -1,5 +1,34 @@
-.tags-container{
+.tags-container {
     display: flex;
-    flex-direction: row;
+    flex-direction: column;
+    align-items: center;
 }
 
+.tags-form {
+    max-width: 220px;
+    background-color: rgb(189, 189, 189);
+    margin: 2px;
+    padding: 10px;
+    display: flex;
+    justify-content: space-between;
+    border-radius: 50px;
+}
+.tags-input {
+    border: 0;
+    background-color: rgba(189, 189, 189, 0);
```

Comment:
`rgba` is for adding opacity, and here `a` is 0

anyway, prefer HEX for consistency

Reply:
done